pypsa/pf.py

```diff
@@ -1320,6 +1336,23 @@ def calculate_Y(sub_network, skip_pre=False):
     )
 
 
+def get_network_ptdf(network):
+    # convenience function
+
```

Comment:
Maybe add a small docstring that this gives PTDF for whole network (not per subnetwork).